storage/local.py

```diff
@@ -34,7 +34,7 @@ def get_content(self, path):
     def put_content(self, path, content):
         path = self._init_path(path, create=True)
         with open(path, mode="w") as f:
-            f.write(content)
+            f.buffer.write(content)
```

Comment:
@arajkumar Actually, why this change?

Reply:
Actually in Python 3 `f.write` expects a string type, however we pass byte. `f.buffer.write` seems to be fine with bytes.

Reply:
@kleesc Can you verify this change?

Reply:
I think it would be better to either open the file in `wb` mode, or decode the content as a `str`, just because of the doc on buffer:
> The underlying binary buffer (a BufferedIOBase instance) that TextIOBase deals with. This is not part of the TextIOBase API and may not exist in some implementations.

Reply:
Thanks @kleesc , Modified to use binary file mode.